build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -137,26 +137,34 @@ def get_git_commit_hash(commit: str) -> str:
 
 
 def get_iree_benchmark_module_arguments(
-    results_filename: str,
     driver_info: DriverInfo,
+    results_filename: Optional[str] = None,
     benchmark_min_time: Optional[float] = None,
+    capture_mode=False,
 ):
     """Returns the common arguments to run iree-benchmark-module."""
 
     if driver_info.loader_name == "vmvx-module":
         # VMVX is very unoptimized for now and can take a long time to run.
         # Decrease the repetition for it until it's reasonably fast.
         repetitions = 3
+    elif capture_mode:
+        # Capture mode is slower and we just need enough repetition to collect
+        # trace after the warmup phase.
+        repetitions = 4
```

Comment:
> however that will increase the total run time too much (>1hr on Pixel 6 benchmarks)

We could prune the set of benchmarks further. Do we have a per-benchmark breakdown of how the 45 minutes in https://github.com/openxla/iree/actions/runs/5870419085/job/15918198968 were distributed?

(fine for follow-up work, agreed that the utility is worth it here)

Reply:
Yeah, I'm also curious about that (and will help prune the benchmarks). Let me do it as a follow-up study